packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -31,11 +31,11 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
       {showSearchInput &&
         (isDesktop ? (
           <div className={styles.patientSearchBar}>
-            <PatientSearchBar hidePanel={() => setShowSearchInput(false)} small floatingSearchResults />
+            <PatientSearchBar hidePanel={() => setShowSearchInput(false)} small orangeBorder />
           </div>
         ) : (
           <Overlay close={() => setShowSearchInput(false)} header={t('searchResults', 'Search Results')}>
-            <PatientSearchBar hidePanel={() => setShowSearchInput(false)} />
+            <PatientSearchBar hidePanel={() => setShowSearchInput(false)} floatingSearchResults={false} orangeBorder />
```

Comment:
Hmm... when should this have an orange border? Not sure when I should be using this

Reply:
This orange border is only visible for the global search. Since I'm using the same bar for global and extensionn works, hence there are a few checks kept. You don't need to use the orange border.